<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QLabel" name="rightNameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="rightMDEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="SHA256Check">
        <property name="text">
         <string>SHA256</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leftSHA1Edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>398</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="leftNameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="SHA1Status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="MD5Status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="MD5Check">
        <property name="text">
         <string>MD5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件名</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="rightSHA1Edit"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leftSHA256Edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="SHA1Check">
        <property name="text">
         <string>SHA1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="SHA256Status">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="selectLeftBtn">
          <property name="text">
           <string>选择左侧文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectRightBtn">
          <property name="text">
           <string>选择右侧文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calcBtn">
          <property name="text">
           <string>计算并比较</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="rightSHA256Edit"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leftMD5Edit">
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>打开</string>
     </property>
     <addaction name="actionOpenLeft"/>
     <addaction name="actionOpenRight"/>
    </widget>
    <addaction name="menuOpen"/>
   </widget>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>配置(&amp;C)</string>
    </property>
    <addaction name="actionEnableMD5"/>
    <addaction name="actionEnableSHA1"/>
    <addaction name="actionEnableSHA256"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>操作(O)</string>
    </property>
    <addaction name="actionCalc"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_C"/>
   <addaction name="menuOperation"/>
  </widget>
  <action name="actionOpenLeft">
   <property name="text">
    <string>左侧文件</string>
   </property>
  </action>
  <action name="actionEnableMD5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>启用MD5</string>
   </property>
  </action>
  <action name="actionEnableSHA1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>启用SHA1</string>
   </property>
  </action>
  <action name="actionEnableSHA256">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>启用SHA256</string>
   </property>
  </action>
  <action name="actionOpenRight">
   <property name="text">
    <string>右侧文件</string>
   </property>
  </action>
  <action name="actionCalc">
   <property name="text">
    <string>计算</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionEnableSHA1</sender>
   <signal>toggled(bool)</signal>
   <receiver>SHA1Check</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnableMD5</sender>
   <signal>toggled(bool)</signal>
   <receiver>MD5Check</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnableSHA256</sender>
   <signal>toggled(bool)</signal>
   <receiver>SHA256Check</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SHA1Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionEnableSHA1</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MD5Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionEnableMD5</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SHA256Check</sender>
   <signal>toggled(bool)</signal>
   <receiver>actionEnableSHA256</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectLeftBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenLeft</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectRightBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenRight</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calcBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionCalc</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
